core/src/main/resources/db/migration.sql

```diff
@@ -191,3 +191,83 @@ CREATE TABLE `structural_variant` (
   FOREIGN KEY (`GENETIC_PROFILE_ID`) REFERENCES `genetic_profile` (`GENETIC_PROFILE_ID`) ON DELETE CASCADE
 );
 UPDATE info SET DB_SCHEMA_VERSION="1.3.1";
+
+
+##version: 1.3.2
+-- ========================== start of genetic_entity related migration =============================================
+-- add genetic_entity table 
+CREATE TABLE `genetic_entity` (
+  `INTERNAL_ID` int(11) NOT NULL AUTO_INCREMENT,
+  `ENTITY_TYPE` varchar(45) NOT NULL,
+  PRIMARY KEY (`INTERNAL_ID`)
+);
+-- update gene table to use genetic_element:
+ALTER TABLE `gene` 
+ADD COLUMN `GENETIC_ENTITY_ID` INT NULL FIRST;
```

Comment:
will ENTREZ_GENE_ID still be the primary key for the table **gene** ?  I think it is usually good practice to keep the primary key fields first in the column order for a table. I'm not sure if that is recommended, but if GENETIC_ENTITY_ID is the first column I personally might expect it to be the PRIMARY KEY unless I look closer.